### PR TITLE
lapack,cgo: rename CompSV to Comp

### PR DESCRIPTION
**Pull request by vladimir-ch**
Partly addresses #160 and #163

### CODE REVIEW

File: lapack.go [hunk 1]
```diff
@@ -10,13 +10,7 @@ const None = 'N'
 
 type Job byte
 
-// CompSV determines if the singular values are to be computed in compact form.
-type CompSV byte
-
-const (
-	Compact  CompSV = 'P'
```
Comment (vladimir-ch) [inline lapack.go]: These two were unused.

**Comment by btracey** [comment]
Good thing you filed a reminder :)

LGTM

**Comment by vladimir-ch** [comment]
I'm also thinking of removing `type SVDJob byte` and using just `Job` for the SVD constants, and similarly use `Comp` for `type EigComp byte`. What do you think?

**Comment by kortschak** [comment]
LGTM

Other changes SGTM

**Comment by btracey** [comment]
Me too. It would be nicer to have a name per type, but Lapack plays pretty loose.

**Comment by vladimir-ch** [comment]
> It would be nicer to have a name per type, but Lapack plays pretty loose.

And distinguishing them in `clapack` would be troublesome.

Thanks.

**Pull request merged**